app/scripts/services/brackets.coffee

```diff
@@ -9,6 +9,13 @@
 ###
 angular.module('edudashAppSrv').service 'bracketsSrv', ->
 
+  getBrackets: (metric) ->
```

Comment:
hmm. I guess we have to give numbers to highcharts? This makes me nervous because

1. The numbers for bracketing are duplicated here
2. Any specification for `<=` vs. `<`; `>=` vs. `>` are lost.

We should try to at least avoid **1**, and make sure these key thresholds are not duplicated in `brackets.coffee`.